spec/requests/users_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+
+RSpec.describe "Sessions" do 
+  new_user = FactoryBot.create :user
```

Comment:
Don't use local variables, use let() https://relishapp.com/rspec/rspec-core/v/2-11/docs/helper-methods/let-and-let